content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -29,7 +29,7 @@ const NewTabPage = React.createClass({
           </section>
 
           <section>
-            <GroupedActivityFeed title="Top Activity" sites={props.History.rows} length={20} />
+            <GroupedActivityFeed title="Top Activity" sites={props.History.rows} length={6} />
```

Comment:
Was this to match a UX spec or prevent v-scrolling or something, ~~or just a speed thing so we don't need to send as many URLs to embedly proxy~~ (nope, I'm still sending 11 `urls=` to embedly proxy, we're just only showing 6 of them)?